docs/review-comments-data.yml

```diff
@@ -0,0 +1,7 @@
+- paths: '**/*.*'
```

Comment:
Let’s do more complex example, at least with 2-3 different checks

Reply:
that's the plan, for now, I want to ensure the basic one works

Reply:
Sorry, I can’t accept PoC in the Slow reader since the PR will be broken.